sdk/communication/azure-communication-chat/src/samples/java/com/azure/communication/chat/CreateChatThread.java

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.chat;
+
+import com.azure.communication.chat.models.ChatParticipant;
+import com.azure.communication.chat.models.CreateChatThreadOptions;
+import com.azure.communication.chat.models.CreateChatThreadResult;
+import com.azure.communication.chat.models.SendChatMessageOptions;
+import com.azure.communication.chat.models.SendChatMessageResult;
+import com.azure.communication.common.CommunicationTokenCredential;
+import com.azure.communication.common.CommunicationUserIdentifier;
+import com.azure.core.http.rest.PagedIterable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CreateChatThread {
+    public static void main(String[] args) {
+        String endpoint = System.getenv("AZURE_COMMUNICATION_ENDPOINT");
+        String token = System.getenv("AZURE_COMMUNICATION_TOKEN");
+        CommunicationTokenCredential tokenCredential = new CommunicationTokenCredential(token);
+
+        // Initialize the chat client
+        final ChatClientBuilder builder = new ChatClientBuilder();
+        builder.endpoint(endpoint)
+            .credential(tokenCredential);
+        ChatClient chatClient = builder.buildClient();
```

Comment:
You can make this is one line like ChatClient chatClient = builder.endpoint().credential().buildClient(). With the correct indentation as you did in lines 24 and 25, it should look good :)

Reply:
Actually, you could create the whole ChatClient without allocating space for the builder and just create the ChatClient like... ChatClientBuilder().endpoint.credential.buildClient()